public/javascripts/Progress/src/MistakeCarousel.js

```diff
@@ -13,12 +13,30 @@ function MistakeCarousel() {
         let labelTypesWithoutData = [];
         labelTypes.forEach((l) => (data[l].length > 0 ? labelTypesWithData : labelTypesWithoutData).push(l));
 
-        // Add subheader for labeling mistakes section, listing label types without validations.
-        let mistakesSubheader = document.getElementById('mistakes-subheader');
+        // This is the list of label types without validations
```

Comment:
Make sure to follow the style for comments in the [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide#general-guidelines)!